File: Modul_5/Bahasa_C/PRAK502-2510817120005-NorKhalisyah.c
#include <stdio.h> 
#include <math.h> 
int hitung(int nilai1, int nilai2){ 
    return nilai1 - nilai2;
} 

int mutlak(int angka){ 
    if(angka < 0)
        return -angka;
    else    
        return angka;
} 
int main() 
{
    int a,b,c,d; 
    printf("Masukkan nilai x1: ");
    scanf("%d",&a);

    printf("Masukkan nilai y1: ");
    scanf("%d",&c); 

    printf("Masukkan nilai x2: ");
    scanf("%d",&b); 

    printf("Masukkan nilai y2: ");
    scanf("%d",&d); 

    int hasil = hitung(a,b) + hitung(c,d); 

    printf("%d",mutlak(hasil)); 
return 0; 
}

File: Modul_5/Bahasa_Python/PRAK502-2510817120005-NorKhalisyah.py
def hitung(nilai1, nilai2):
    return nilai1 - nilai2

def mutlak (angka):
    if angka < 0:
        return -angka
    else:
        return angka

a = int(input("Masukkan niali x1: "))
b = int(input("Masukkan niali y1: "))
c = int(input("Masukkan niali x2: "))
d = int(input("Masukkan niali y2: "))

hasil = hitung(a, c) + hitung(b, d)
print(mutlak(hasil))